ninja-servlet/src/main/java/ninja/servlet/ContextImpl.java

```diff
@@ -134,6 +151,16 @@ public void setRoute(Route route) {
     }
 
     @Override
+    public void cleanup() {
+        // no op for non-multipart request context
+        for (List<NinjaFileItemStream> items : fileItems.values()) {
```

Comment:
please remove thespacing between : and items (we want to keep the formatting uniform... :) )